### ISSUE TITLE
1 yearssss we hadddd meeettttt or 1st talk ehhhhhhhh

### ISSUE THREAD

**Issue by BushraOnika** [IAFahim/HackerRank-SQL#1]
Eid Mubarak🤗🥺
Tataaaa

**Comment by IAFahim** [comment]
You remembered our friendiversary!! How is this even Bushra.
Yes 23 years old naa, one year me disturbed you, yeahyy. Won't give back huh. Nice nice and sweet Oniy. Eid Mubarak to you too. Take care or I give boka in my to Oniy who is in my brain, lots of boka. And you study also, make lots of friends and and study study. And me pray you never get sad sad ever again. Go go give time to family, kalmoni and her little soliders. And code a lot also. And I trust you would find the best material to study and be someone super important in this world.

**Comment by IAFahim** [comment]
> Yesss you always forgot so me had to remember.Me remember everything huh😝

So much talent you have, wow. You remembered see you still think about me, yeahyy. Or you have good calendar which is also nice.

And about responsibilities that you will have, everyone have, that's why we can not worry sometimes and have rest and lot's of fun with others. And you should make friends silly. University should be best time of our life right. Or how you tell little Oniy's about all your friends. 😗 Not like super cool like me, who already have bestest friend Onion roaming NSU campus all time.
Won't block this issues. You can text when you are angry at me. Angry angry onion.

**Comment by BushraOnika** [comment]
No I don't have calendar huh me remember I have a sharp brain which stores everything.😝😝
Eeehhh how you know me roaming NSU maybe you had seen ghosts who knows lots of ghosts in NSU 👻🧐🐸
You won't block this one wow how did this magic happens 😱😱
Do you have fever or you feeling sick !!!!!🤒

**Comment by IAFahim** [comment]
Such good brain of yours.
No it was you once I wasn't sure so me came close then though ahh I she would give boka so me opened umbrella and went into rain.
Yes yes I won't close but you shouldn't chat naa, you you should give time to hubby only naa. He would be jealous you would see. How he share pretty Bush huhhh. And you aren't allowed to feel sad or bad bad. You are sweet naa, sweet people should be happy. You solve this problems too, would help. (Collecting rain water with so white and pretty hand I should give boka and someone should give you pitty too) go go spend time with Ammu.

**Comment by BushraOnika** [comment]
> No it was you once I wasn't sure so me came close then though ahh I she would give boka so me opened umbrella and went into rain. Yes yes I won't close but you shouldn't chat naa, you you should give time to hubby only naa. He would be jealous you would see. How he share pretty Bush huhhh. And you aren't allowed to feel sad or bad bad. 

O yes me remember you hada green umbrella me thought you were ignoring me huh.
You said when I'm angry with you me can text here and you won't block so me angry with you so me can text now.and and you can't block you said only When me won't be angry me won't text huh.

**Comment by IAFahim** [comment]
Oh no so angry Onika. Don't be angry. You are always happy but me still say, don't know if you get sad sometimes naa. Sorry for making you feel like you were ignored. You were also my bestest friend too, so boka youu and pretty offoh oh. Sleep sleep, tata.

**Comment by BushraOnika** [comment]
No need to say sorry you ignored me then me also ignore you On Wednesday yes me 1st forgot to take revenge so me called you but then me remember you ignore me that day so me also did and took my revenge huh.
Now me become calm so me won't text you. But you won't close this comment you had promised. Cause when me will be angry again me will tell here. Tata for now. 😝

**Comment by IAFahim** [comment]
So there is revenge you too. I couldn't understand how you took revenge but knowing you it was something silly, as you are silly too hehe. Hay, me just not disturb you. You awake so late night hope you were studying. Tata please become someone extremely successful or really happy in life or both would do also. Good that we talked last few times as usual thanks to nice Oniy. Thank you for taking care of me and not letting others know how your exactly feeling and always being Oniy for them should be very hard. So you take care of youu okay. Tata tata.

**Comment by BushraOnika** [comment]
So rude you didn't ask the reason also seee. Rude stranger as always. No me took vacation on Eid days no study at all 3 days are Eid so no study on 3days huh 😝. But why thanking me I never took care of youuuu🤷‍♀️
And becoming successful ufffo I had promised someone so I have to fulfill that me know so much hard work. 😷
And about expressing my feelings that I only do this my close peoples so others would never get the hints also how I feel cause I'm like that only so no hard thing for mee at all cause from child I'm like that.

**Comment by IAFahim** [comment]
> So rude you didn't ask the reason also seee. Rude stranger as always.

No no, didn't ask because its Tata tata text naa, thats why. And me pocha always you forgot huh.
> No me took vacation on Eid days no study at all 3 days are Eid so no study on 3days huh 😝

now vacation over right you can study again, yeahy.
>But why thanking me I never took care of youuuu🤷‍♀️

Youuuu did you just don't know because you nice and don't notice that other people doesn't do. So you don't know. Study study and never change. Good you.

> And becoming successful ufffo I had promised someone so I have to fulfill that me know so much hard work. 😷

you promised someone? Then ok, thought had to give lots of boka, but now you are safe.

> And about expressing my feelings that I only do this my close peoples so others would never get the hints also how I feel cause I'm like that only so no hard thing for mee at all cause from child I'm like that.

Hope when you get big Oniy then you have lots of close people that you can tell, they would like to know naa. Cant give boak to child Oniy too. She is pretty too i can see in profile pic, huh huh.

Now you go, study time has come for you. tata. And do I have to ask reason hmm you are a lot thoughtful and wise. You tell when you trust someone naa, so when you trust me you would have told and do lots of bokor bokor too which is the best thing others want na. now you go.  No more talking need to fulfill promises. Go study. Do your bestest oniyyyyyy.

**Comment by IAFahim** [comment]
> And you could have asked the reason naaa but you didn't huh soooo Me won't be bestest huh. Tataaaa won't talk.

Nope, won't ask, it's okay, I trust you naa. If you study, then not talking is okay.

**Comment by BushraOnika** [comment]
> > Seee seeee you said me always do bokor bokor.
> 
> No boka, I told you should do bokor bokor. You don't tell na how you feel so I told.

O accha accha me just become confused little huh me know me know me always calm and silent so me get angry why you say me always do bokor bokor. Now its ok 😝

**Comment by IAFahim** [comment]
> Go go you study me won't huh. Me on Eid vacation.

Ow yes resting you is also ok

**Comment by IAFahim** [comment]
> O accha accha me just become confused little huh me know me know me always calm and silent so me get angry why you say me always do bokor bokor. Now its ok 😝

Yeahy all matter resolved. Still silent and calm you.

**Comment by BushraOnika** [comment]
Yes silent and calm and you said me silly but opposite of silly is me.

**Comment by IAFahim** [comment]
Silly, wisdomus. huh huh.

**Comment by IAFahim** [comment]
I know you're awesome, everyone knows. Now rest please you are on vacation.

**Comment by IAFahim** [comment]
boka

**Comment by IAFahim** [comment]
Not bot boka happy 😗
You confident too
Go go

**Comment by BushraOnika** [comment]
> Not boka boka happy 😗 You confident too Go go
Now its sounds good 😌
No no me won't go anywhere me loves my home and missed it a lot so me will stay there only huh

**Comment by IAFahim** [comment]
So easy to make you boka.
Youu won't imagine you in home at all, you have all Oniy pics and puter full of more you. But Mithu would be disturbing you so it's nice nice. You study study at home it's already new day so you can study and no one would disturb you also 🙂 finally you would have everything to be wise. Aww you started to making me flash back of all the punishment and you. Tata me busy with brain Oniy why me talk with you. Sleeep.

**Comment by BushraOnika** [comment]
Seeee again you said boka 😤
Say not bokaaa. Nope its not easy to make me boka me clever girl always huh. 
Tata me sleeppp 😝

**Comment by IAFahim** [comment]
Chup you sleep. And do SQL practice tomorrow. Project due in month.

**Comment by IAFahim** [comment]
And you are never boka, you are just Oniy

**Comment by BushraOnika** [comment]
> Chup you sleep. And do SQL practice tomorrow. Project due in month.

Ewwww the best thing is I have no projects this semester yeeeeee😍
And I don't know what is sql!!!!!🙈

**Comment by BushraOnika** [comment]
O sorry one project but sir didn't said anything except he will give project so me safe for now huh

**Comment by IAFahim** [comment]
You didn't take cse311?

**Comment by IAFahim** [comment]
You can't sleep? 😗 Thinking about me too much?

**Comment by IAFahim** [comment]
> O sorry one project but sir didn't said anything except he will give project so me safe for now huh

Which course is it?

**Comment by BushraOnika** [comment]
> > O sorry one project but sir didn't said anything except he will give project so me safe for now huh
> 
> Which course is it?

Cse323

**Comment by IAFahim** [comment]
You bought Arduino board?

**Comment by BushraOnika** [comment]
> You bought Arduino board?

Noooooo

**Comment by IAFahim** [comment]
Is it okay if I say you are the most prettiest girl in NSU? I have no idea why you talk with me even after I am always rude with you. You should talk with someone nice naa. Instead you texting someone who you only meet like 20 times. And had maybe 10 conversation in person. I can't be around you naa. No talk you you go silly. I don't want waste Your time, read something, make something, your whole life is ahead of you and you waste time with something that won't be part of your life. Not fair at all. And am going somewhere where there is lots of Arduino board I would call you if I am there and if you need something. Tata

**Comment by BushraOnika** [comment]
The thing is I missed you a lot. But ok tataaa. I don't need Arduino board cause I have no idea about that at all. Tataaaaa

**Comment by IAFahim** [comment]
I miss you a lot too na, but it's not my place to miss you. I know it doesn't make it better. But we talked for a year naa. So it's interesting in a way that we didn't treated each other that good and now when we don't talk we miss. But that has lot of pain naa. Talking with someone who isn't part of life. I should have been better listener with you then you wouldn't have felt so bad naa. Sorry for being so stupid. Youu go you weren't made for me, you would belong to someone else. So you shouldn't be sad. It was already planned now you should be happy only. Don't want you be sad naa. Ehh you closed while I was typing. Means you not angry at me anymore. You go study, you would have someone supa nice in your life if you don't look for him how would he come he must be shy shy too. Tata Bushra Hossain Onika. See I was talking with you everyday in head now I get to say goodbye to you again. Don't feel any good the first time at all only thing would make me happy if you fullfil all your dreams. And I root for you from super far distance. Been a lot horrible to you and that was me trying my best not that good at all. Please be happy in life. And be Oniy with everyone you meet. Everyone would like you I see how other girls like you hmm hmm see you are bestest. Now you go won't get to hurt you ever again. Tata

**Comment by IAFahim** [comment]
I would miss you. Hope my heart doesn't hurt everytime I think about you, Allah Hafez.

**Comment by BushraOnika** [comment]
Ewww seee who was saying or giving lecture about making new friends to meeeeeeeeeee ehhhhhh!!!!! 
Youuuuuuu should make new friends. A cute who knows graphic design welll and most talented friend huh.ohoooo we should have more cute cute new members in team promotion. Wait next time we will add cute cute new members huh. 
And and unfollow me from GitHub or how can you forgot huh.

**Comment by IAFahim** [comment]
![Screenshot_20220713-152205~2](https://user-images.githubusercontent.com/63500913/178699884-e352e9b6-244f-4b5a-afd2-62e6cafe0228.png)

Shu shu go no texty

**Comment by BushraOnika** [comment]
O yes me remember your cse323 teacher also likes me huh. 
Are you going by Padma setuuuu yeeeeeeee have safe journey....
Ohooo you and your memories ufff my kacchi is going to missed no worries me will call your ammu and will said to get your married soon huh in case you don't invite me, me would come to have kacchi😌
Happy youuuu 😍
And I didn't understand the meme how can a people became vector analysis🙂

**Comment by IAFahim** [comment]
Nope visiting plots. You you can have kacci at home. You you make good chef you. Not getting married at all girls are stupid huh. 
> And I didn't understand the meme how can a people became vector analysis🙂

Ow wait nice point where did he go, They study about it. It meant that. Go you study too. Tata

**Comment by BushraOnika** [comment]
Your cse teacher is my CSE and EEE teacher 2 in one so me know huh 🙊🙈
You know na me horrible chef so you should get married me will eat kacchi na🥺
And and girls are sweet cute and talented all girls are not like me stupid huh. So you would marry a sensible girl then you would be happy happy always ohoo😌
Seee after studying he transformed into vector analysis no me won't study me don't want to be vector analysis huuuuusssshhhhhhhh.

**Comment by BushraOnika** [comment]
Nope boka means stupid also so me know huh. 
And don't get married ok me also won't if you don't tataaaaaaaaaa

**Comment by IAFahim** [comment]
You want to not let me have kacci at your wedding because me told won't have right 🥺 you can have my part kacci then. Remember you told you would marry someone Abbu picks and me told Abbu would pick the best person so it would be fine. Now can't change. But actually if you don't get married then you would have more time to study and travel, and you want to go Islamic country and Seattle there right? With hubby it would be much easier. If you study and become good then also can do. But no copying me huh. Never ever seen you serious at all na so I am scared. Wait let me give boka so that you won't be able to copy me. Can't let you destroy your happiness just because of strange. I plan to hurt you a lot in this few text so you never speak with me again. 
 
Me don't like you at all if I liked you we would be talking of course. 
you you texty texty and not study.
You should study study and do no text.
I am not good. Listen our brain is more claver then us, when a teacher is good everyone likes them right and they treat them nice nice? So the reason student likes the teacher is not because all students were good it's because teacher is good. So we only see our reflection in others so if you are a good person everyone would try to act nice to you. See that's why everyone likes you, not because everyone is good but you you, it's your refection. So duh I did too because you told me how you were and I get to know you little little. Now see you you don't like to talk with me when you are angry or busy but in truth that's the exact moment when people are most honest, you always nice nice naa so I thought yeahy me pocha she says but must be little good right or why she spend little time with me. But I forgot it wasn't that naa you are super polite so you make other feel important, but how would you always keep it up silly. And I couldn't understand and always used you to make myself feel good. Lots of selfish. See like that day if someone unknown came and talked with you, you would have given a little time right then get busy busy again. And I had no idea how bad I was naa, I don't even know how a person could be less then a stranger, that don't want to talk even. Tell me the truth do you think I would ever helped you if you were rude or not good looking person, ok maybe I didn't do it to have you as friend or maybe I did. See always disturbing you for pics of you. See the connection, oh yess it was all a plan to trap you and make you super unhappy by taking all your time because of my evil plan. But then I saw that how little I was I understood no I can't take fool her she knows so I left, all part of plan.

Biggest boka ever. Now see wise you saw though everything now your time to judge what you would do, waste more time here writing or fulfill your dream by studying hard your choice. To make it more painful I can close the issue and make you feel more bad, but I won't because it is also part of plan that you would understand everything is a lie I told and you would text again but maybe it's not my plan. But one thing I made sure you understand crystal clear I hope you are the good person and did nothing wrong but to be yourself. Tata think of this issue as closed just to make it more painful for you. Leave, but study btw and code and push to GitHub, need for job. So where were I, oh yes leave. And you can never know if I get married or not after NSU think about it you can't actually find me. So like think I did 20 marriage with all the perfix of your name girl like your name is "Bushra Hossain Onika" which takes 20 char so for each char I married someone named with char 
'B' -> Bash
'U' -> Udemy
'S' -> Scala
...
Even for space also huh (eg: ' ')
You can find it funny and leave or or you can be like why he joke in so serious matter and leave. But remember this
Y
O
U

A
R
E

Bestest 😉

No destroying own life of changing it. So what if I like you or not like you. Be happy that cool cool me though you were the most nicest person I ever meet. Should be something to be happy about. 😗 Little narcissism to end it to have no recovery from this text. Dam I am self destructive I mean supa cool.

**Comment by BushraOnika** [comment]
Don't you dare to say anything bad about my stranger. I have never give anyone(including you) the right to say anything bad about him in front of me and only I have the right to say anything about him and I don't like sharing my thinks so don't dare. 
And about my marriage you don't have to worry like you said I would never know after nsu if you had married or not,like that you also won't never know so think that I'm happily married the problem is solve. 
And don't dare to say anything about stranger I don't give my rights to anyone (including you) so don't. 
And yes don't text anything here cause I won't listen to you anyways don't waste your time here convincing me. Cause if I decide something I never listen to my parents also so if you don't know let me say I'm the most ziddi person in my family. 
So don't waste your energy or time here by texting long long text go study it will help you.

**Comment by IAFahim** [comment]
> Don't you dare to say anything bad about my stranger.

Youuuuu 🥺 you need pitty huh
> I'm the most ziddi person in my family.

You missed most beautiful but me allow your Ammu is also good but ehh you win. If you are ziddi then be someone super wise then I would believe you. 
> So don't waste your energy or time here by texting long long text go study it will help you.

Ow yes yes good advice, youu you no calling it waste, why would talking with you be waste. But me go to study. Need to be good boy naa. Tata tata.

**Comment by IAFahim** [comment]
![qrd07ic5wfb91](https://user-images.githubusercontent.com/63500913/178982124-b34d1971-bb14-4a41-8702-00928a0be8b4.jpg)

We couldn't be lab partner noooooo...

**Comment by BushraOnika** [comment]
Youuu told you na don't text and you are sending memes 😒
Its good that we don't become lab partners then you wouldn't got the chance to leave😂
You will always be my favourite person, even if we are not friends anymore! 💜
And and don't text go away i got new new good friends go shu shuuuuuuuuu huh

**Comment by BushraOnika** [comment]
![FB_IMG_1657910267828](https://user-images.githubusercontent.com/87435687/179289544-830f8915-41a0-4ce7-95e9-4b2cf50effd9.jpg)

Tataaaaaaa don't reply anything.

**Comment by IAFahim** [comment]
> Youuu told you na don't text and you are sending memes 😒 Its good that we don't become lab partners then you wouldn't got the chance to leave😂 You will always be my favourite person, even if we are not friends anymore! 💜 And and don't text go away i got new new good friends go so don't think about me now shu shuuuuuuuuu huh

Oh yes yes I send meme to disturb huh, too peaceful not to disturb. And you not leaving would be nice naa, see you need lab partners lots of them. Oh yes you got new Friends("supa nice kind") so don't need to disturb at all they would disturb themselve, my task is done, yeahyy. Now have to do less thinky thinky of you and done tadaa. Going going baba. Tata

**Comment by IAFahim** [comment]
> ![FB_IMG_1657910267828](https://user-images.githubusercontent.com/87435687/179289544-830f8915-41a0-4ce7-95e9-4b2cf50effd9.jpg)

Didn't give pitty huh so angry, angry me know 😗 you were like why haven't he give any boka so rude.
> Tataaaaaaa don't reply anything.

Like I would listen, tata

**Comment by IAFahim** [comment]
> You will always be my favourite person

And I will always remember you and won't give you any kisses at all, tata 😗 you
